### DISCUSSION TITLE
How to create empty module template?

### DISCUSSION THREAD

**Discussion by kosthetagurov** [oqtane/oqtane.framework#1452] [Show and tell]
![image](https://user-images.githubusercontent.com/45827311/120981872-c59bde80-c780-11eb-83cf-a2ff3294a20b.png)

Is it possible to create custom (emty to example, no CRUD by default) templates when i create module, if i only need use oqtane infrastructure?

**Comment by vzdesic** [comment]
Yes, it is possible. 

Copy "~\oqtane.framework\Oqtane.Server\wwwroot\Modules\Templates\External" folder and delete all what you don't need inside your external module. If the name of your folder will be "Empty" for example, than you will be able to select it in the Template dropdown, not only "External".
